services/web/server/src/simcore_service_webserver/login/utils.py

```diff
@@ -184,27 +185,33 @@ async def send_mail(app: web.Application, msg: MIMEText):
     smtp_args = dict(
         hostname=cfg.SMTP_HOST,
         port=cfg.SMTP_PORT,
-        use_tls=cfg.SMTP_TLS_ENABLED,
+        use_tls=cfg.SMTP_PROTOCOL == EmailProtocol.TLS,
+        start_tls=cfg.SMTP_PROTOCOL == EmailProtocol.STARTTLS,
     )
     log.debug("Sending email with smtp configuration: %s", pformat(smtp_args))
     if cfg.SMTP_PORT == 587:
         # NOTE: aiosmtplib does not handle port 587 correctly
-        # plaintext first, then use starttls
         # this is a workaround
         smtp = aiosmtplib.SMTP(**smtp_args)
-        await smtp.connect(use_tls=False, port=cfg.SMTP_PORT)
-        if cfg.SMTP_TLS_ENABLED:
-            log.info("Starting TLS ...")
-            await smtp.starttls(validate_certs=False)
+        if cfg.SMTP_PROTOCOL == EmailProtocol.STARTTLS:
+            log.debug("Unencrypted connection attempt to mailserver ...")
+            await smtp.connect(use_tls=False, port=cfg.SMTP_PORT)
+            log.debug("Starting STARTTLS ...")
+            await smtp.starttls()
+        elif cfg.SMTP_PROTOCOL == EmailProtocol.TLS:
+            await smtp.connect(use_tls=True, port=cfg.SMTP_PORT)
+        elif cfg.SMTP_PROTOCOL == EmailProtocol.UNENCRYPTED:
+            log.debug("Unencrypted connection attempt to mailserver ...")
```

Comment:
you do not need to connect here? ``smtp.connect(use_tls=False, port=cfg.SMTP_PORT)``?